app/services/support_interface/candidate_email_send_counts_export.rb

```diff
@@ -0,0 +1,24 @@
+module SupportInterface
+  class CandidateEmailSendCountsExport
+    def data_for_export
+      emails_with_counts.map do |email_hash|
+        {
+          'Email' => email_hash['mail_template'],
+          'Send count' => email_hash['send_count'],
+          'Last sent at' => email_hash['last_sent_at'],
```

Comment:
Do we specify the time format?

Reply:
Just using the UTC timestamp straight from the db at the moment.